server.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+const express = require('express');
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+const webpack = require('webpack');
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('./webpack.config.js');
+
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 3000.
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Example app listening on port 3000!\n');
+});
```

Comment:
- [ ] I am unable to run your app. Kindly ensure the app is properly set up with webpack. I should be able to run commands like `npm install`, or `npm start`/`npm run build` to compile webpack and start the application.

  If the app does not use the above commands then kindly list the detailed steps on how to set up and run the app locally on your README. I could not find instructions on how to do that.